isisd/isisd.h

```diff
@@ -63,6 +63,7 @@ extern struct zebra_privs_t isisd_privs;
 struct fabricd;
 
 struct isis {
+	vrf_id_t vrf_id;
```

Comment:
I think we need to have a `vrf` pointer here instead, and use the libfrr VRF hooks to update this pointer whenever a VRF is enabled or disabled. Having to do a VRF lookup whenever necessary is ugly in my opinion.

Reply:
yeah we can remove this commit, it's the start of a long line of moving over to the vrf imo as a transitional commit I think it's ok as a long term solution I agree I think.